src/project_manager/ros_utils.cpp

```diff
@@ -706,6 +761,28 @@ Utils::FileName ROSUtils::getCatkinToolsProfilesYamlFile(const Utils::FileName &
     return profiles;
 }
 
+bool ROSUtils::setCatkinToolsProfilesYamlFile(const Utils::FileName &workspaceDir,
```

Comment:
Can you use this [function](https://github.com/ros-industrial/ros_qtc_plugin/blob/master/src/project_manager/ros_utils.h#L336).